zmon_worker_monitor/worker.py

```diff
@@ -4,6 +4,8 @@
 """
 
 import settings
+import logging
+from .tracing import init_opentracing_tracer
```

Comment:
I would suggest also adding:

```python
from opentracing_utils import trace_requests
trace_requests()
```
so that all outgoing requests will propagate the trace/span IDs

Reply:
done